### PR TITLE
PR to track changes for v5

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: src/GraphQL.Authorization/GraphQL.Authorization.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL" Version="4.6.0" />
+    <PackageReference Include="GraphQL" Version="5.1.1" />
```
Comment (sungam3r) [inline src/GraphQL.Authorization/GraphQL.Authorization.csproj:10]: @Shane32 Bump to 5.2 or not?

Comment (Shane32) [reply src/GraphQL.Authorization/GraphQL.Authorization.csproj:10]: There is nothing in 5.2 over 5.1.1 that relates to authentication.

**Review by Shane32 — APPROVED** [review]
Good enough.

**Comment by Shane32** [comment]
I would like to have seen the Skip condition code as a separate PR, but there's many other parts of this entire project that I would redesign, and besides, we already have two other authentication projects.  I don't plan on using this project at all.  It may be useful when hosting GraphQL outside of ASP.NET Core, however.

**Comment by sungam3r** [comment]
At least it should be published to be in sync with the latest GraphQL.NET.

**Pull request merged**